app/models/course_data/articles_courses.rb

```diff
@@ -100,7 +101,7 @@ def self.update_all_caches(articles_courses)
 
   def self.update_from_course(course)
     course_article_ids = course.articles.where(wiki: course.wikis).pluck(:id)
-    revision_article_ids = get_mainspace_revisions(course.revisions).distinct.pluck(:article_id)
+    revision_article_ids = course.revisions.distinct.pluck(:article_id)
```

Comment:
Does this change mean that ArticlesCourses records will be created for all namespaces for which revisions exist, regardless of whether it's a tracked namespace?

Maybe there's a way to use `revisions` scopes to get just the revisions from the namespaces we care about? Or maybe this needs to be done by looping through wikis/namespaces? I think we'll need to avoid creating ArticlesCourses records for non-tracked namespaces.